docs/docs/apis/actions/introduction.md

```diff
@@ -10,7 +10,8 @@ ZITADEL interpretes the scripts as JavaScript.
 Make sure your scripts are ECMAScript 5.1(+) compliant.
 Go to the [goja GitHub page](https://github.com/dop251/goja) for detailed reference about the underlying library features and limitations.
 
-You can find great snippets and examples which show how you can use actions in [this repository](https://github.com/zitadel/actions).
+If you need some inspiration on how to set OIDC claims and SAML attributes during token creation, extend information during just in time provisioning from identity providers or how to call an external API and other examples,
+you can find great snippets which show how you can use actions in [this repository](https://github.com/zitadel/actions).
```

Comment:
A more engable alternative to your proposal:

Stuck customizing ZITADEL actions? Find samples for setting OIDC claims, SAML attributes, extending JIT provisioning data, calling external APIs, and more in [this repository.](https://github.com/zitadel/actions)